### ISSUE TITLE
How to add this package to azure hdinsight spark and work for livy.

### ISSUE THREAD

**Issue by Sam--Shan** [crealytics/spark-excel#224]
Hi guys,i use the azure hdinsight to build spark cluster and i need to deal with some xlsx/xls file in my pyspark code. I use livy for submit my application,but i dont know how to add this package for my job.i am new in spark.

## Expected Behavior
 _java.lang.ClassNotFoundException: Failed to find data source: com.crealytics.spark.excel. Please find packages at http://spark.apache.org/third-party-projects.html_

## Current Behavior
submit with livy, pyspark code in azure blob and i want to write my result data back to azure blob as format xls/xlsx.

## Possible Solution
change spark config CLASSPATH in ambari 
add ‘jars’ attribute to livy post data. 
add spark session set
it is still not work

can you guys show me the right way? thanks a lot

## Your Environment
HDInsight spark 2.4(HD4.0)
pyspark

**Comment by jesteban2** [comment]
Hi, I have same issue, could you solve it?

**Comment by Sam--Shan** [comment]
> Hi, I have same issue, could you solve it?

no,i didnt make it with pyspark in hdinsight, and i change csv to xlsx in local machine after i download the csv from blob.i still wrote the result as csv format in blob. Maybe you can try it with scala,i dont kown scala.
